services/QuillLMS/client/app/bundles/admin_dashboard/containers/AdminDashboard.jsx

```diff
@@ -98,27 +97,6 @@ const AdminDashboard = ({ adminId, accessType, passedModel, }) => {
     );
   }
 
-  function resendLoginDetails(data) {
```

Comment:
Was this supposed to get deleted?

Reply:
as far as I can tell, this isn't being used anywhere-- it was just dead code so I got rid of it